src/current/_includes/v24.1/known-limitations/physical-cluster-replication.md

```diff
@@ -1,5 +1,7 @@
-
+- Physical cluster replication is supported only on CockroachDB {{ site.data.products.core }} in new clusters on v23.2 or above. Physical Cluster Replication cannot be enabled on clusters that have been upgraded from a previous version of CockroachDB.
```

Comment:
Please follow up with @kathancox on this outside of this PR, I think the language needs a tweak for v24.1, which allows upgraded customers to enable PCR. I don't want to block this PR. At the same time I am not sure why your PR is changing this line.

Reply:
I think this was a faulty rebase on my part. I'm going to manually replace it with the current published file contents. Thanks for finding this.